npm/src/browser.ts

```diff
@@ -249,3 +249,5 @@ export type {
 } from "../lib/web/qsc_wasm.js";
 export { type IStructStepResult, StepResultId } from "../lib/web/qsc_wasm.js";
 export { type LanguageServiceEvent } from "./language-service/language-service.js";
+
+export * as utils from "./utils.js";
```

Comment:
just to confirm that this export is required

Reply:
If we're going to include it in the main package code (e.g. so we can test it), it should be exposed somehow. For example, if you look at the unit tests now, it imports this export.